latex-template/programme.bib

```diff
@@ -10,10 +10,10 @@ @article{matplotlib
   url = "http://matplotlib.org/",
   version = "1.4.3",
   doi       = {10.1109/MCSE.2007.55},
-  addendum = {Current version 3.4.3, \textsc{doi}: \href{https://zenodo.org/record/5194481}{\texttt{10.5281/zenodo.5194481}}}
+  addendum = {Current version 3.7.2, \textsc{doi}: \href{https://zenodo.org/record/8118151}{\texttt{10.5281/zenodo.8118151}}}
```

Comment:
Current/latest release is 3.8.0, released Sept 13, so I think we should add that here

Reply:
In my latest install this was the version installed.